Makefile

```diff
@@ -65,6 +65,9 @@ endif
 ifeq ($(QUAI_STATS),true)
 	BASE_CMD += --quaistats ${STATS_NAME}:${STATS_PASS}@${STATS_HOST}
 endif
+ifeq ($(TRUSTED_NODE),true)
+	BASE_CMD += --trustednode
```

Comment:
Please explain what 'trusted node' means. Everyone trusts their own node. Nobody should trust anyone elses node. What does this flag indicate?

Reply:
This is for Quai nodes that are running using the admin name. Node Wranglers send block append time and node stats (ram usage, cpu usage, etc). Trusted nodes (operated by quai) send transaction stats (tps) and block details (difficulty, heights).

Trusted nodes require a different password